### PR TITLE
Update Fielmann AG (fielmann-de)

### PR DESCRIPTION
**Pull request by WebworkrNet**
This dataset can be marked as tested.

### CODE REVIEW

File: companies/heise-de.json [hunk 1]
```diff
@@ -4,14 +4,31 @@
...
-    "phone": "+49 511 53520",
-    "fax": "+49 511 5352129",
     "email": "datenschutz@heise.de",
     "web": "https://www.heise.de",
     "sources": [
         "https://www.heise.de/impressum.html",
-        "https://www.heise.de/Datenschutzerklaerung-der-Heise-Medien-GmbH-Co-KG-4860.html"
+        "https://www.heise.de/Datenschutzerklaerung-der-Heise-Medien-GmbH-Co-KG-4860.html",
+        "https://www.heise.de/Individuelle-Datenschutzinformationen-fuer-heise-Academy-und-heise-Academy-Events-4058183.html",
+        "https://www.heise.de/Individuelle-Datenschutzinformationen-fuer-heise-https://www.heise.de/Individuelle-Datenschutzinformationen-fuer-Heise-Business-Services-7099857.html",
```
Comment (mal-tee) [inline companies/heise-de.json:25]: this link is broken!

**Review by mal-tee — CHANGES_REQUESTED** [review]
Thank you, I left a note.

**Comment by WebworkrNet** [comment]
> I left a note.

Unfortunately I don't understand what you are referring to.

File: companies/heise-de.json (hunk 1 above)
Comment (WebworkrNet) [reply companies/heise-de.json:25]: https://www.heise.de/Datenschutzerklaerung-der-Heise-Medien-GmbH-Co-KG-4860.html is fine for me.

Screenshot_20231017_003403_Firefox Nightly

**Comment by baltpeter** [comment]
He's referring to the comment (that you already replied to).

**Comment by baltpeter** [comment]
These changes have since all been merged through different PRs/commits, so we can close this.

**Pull request closed without merging**